packages/smart-wallet/test/gameAssetContract.js

```diff
@@ -48,13 +48,17 @@ export const start = async zcf => {
 
     totalPlaces(want.Places) <= 3n || Fail`only 3 places allowed when joining`;
 
-    atomicRearrange(
```

Comment:
btw... @turadg another static typing nit: `getCopyBagEntries` returns `any`.

Reply:
for https://github.com/Agoric/agoric-sdk/issues/7507